person.rb

```diff
@@ -24,7 +24,9 @@ def correct_name
   end
 
   def add_rentals(book, date)
-    @rentals << Rental.new(date, book, self)
+    @rentals = Rental.new(date, book, self)
+    # @rentals << rental
+    # book.add_rental(rental)
```

Comment:
- [ ] Please kindly removed commented lines of code from your codebase. It makes your codebase clean and professional. Also, you can always use git history for old codes